tutorials/setup.md

```diff
@@ -130,6 +130,63 @@ make BUILDTAGS='seccomp apparmor'
 | exclude_graphdriver_overlay      | exclude overlay as a storage option             | <none>       |
 | ostree                           | build storage using ostree                      | ostree       |
 
+## Static builds
+
+It is possible to build a statically linked binary of CRI-O by using the
+provided [nix](https://nixos.org/nix) package [within this repository](../nix).
+The builds are completely reproducible and will create a `x86_64`/`amd64`
+stripped ELF binary for [glibc](https://www.gnu.org/software/libc) and [musl
+libc](https://www.musl-libc.org). These binaries are integration tested on
+travis as well and support the following features:
```

Comment:
I don't know if `on travis` is really necessary. The implementation of our tests don't really matter to an end user, just that they are actually tested.

Reply:
Yep, removed it.